2020-11-04 21:14:59,465 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-04 21:15:07,120 INFO __main__ VS-rad-median ball_within_template 0 || loading data for generative training...
2020-11-04 21:16:49,242 INFO __main__ VS-rad-median ball_within_template 0 || training 'generative' models...
2020-11-04 21:16:52,325 INFO root training epoch 1...
2020-11-04 21:17:44,304 INFO root collecting auccess after epoch 1...
2020-11-04 21:17:51,379 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-04 21:17:53,126 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 261, in <module>
    solver.train_supervised(epochs=epochs, train_mode=train_mode, test_ids = test_ids, setup=f'{eval_setup}_{fold_id}')
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 1963, in train_supervised
    auccess = self.generative_auccess(test_ids, setup)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 897, in generative_auccess
    if self.radmode=="old":
AttributeError: 'FlownetSolver' object has no attribute 'radmode'
2020-11-04 21:31:49,345 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-04 21:31:57,409 INFO __main__ VS-rad-median ball_within_template 0 || loading data for generative training...
2020-11-04 21:33:39,532 INFO __main__ VS-rad-median ball_within_template 0 || training 'generative' models...
2020-11-04 21:33:42,275 INFO root training epoch 1...
2020-11-04 21:34:09,414 INFO root collecting auccess after epoch 1...
2020-11-04 21:34:16,261 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-04 21:34:18,012 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 261, in <module>
    solver.train_supervised(epochs=epochs, train_mode=train_mode, test_ids = test_ids, setup=f'{eval_setup}_{fold_id}')
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 1963, in train_supervised
    auccess = self.generative_auccess(test_ids, setup)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 897, in generative_auccess
    if self.radmode=="old":
AttributeError: 'FlownetSolver' object has no attribute 'radmode'
2020-11-04 22:26:36,672 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-04 22:26:44,551 INFO __main__ VS-rad-median ball_within_template 0 || loading data for generative training...
2020-11-05 02:11:41,422 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-05 02:11:45,569 INFO __main__ VS-rad-median ball_within_template 0 || loading data for generative training...
2020-11-05 02:13:00,525 INFO __main__ VS-rad-median ball_within_template 0 || training 'generative' models...
2020-11-05 02:13:02,478 INFO root training epoch 1...
2020-11-05 02:13:21,625 INFO root collecting auccess after epoch 1...
2020-11-05 02:13:26,905 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-05 02:13:28,074 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 261, in <module>
    solver.train_supervised(epochs=epochs, train_mode=train_mode, test_ids = test_ids, setup=f'{eval_setup}_{fold_id}')
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 1965, in train_supervised
    auccess = self.generative_auccess(test_ids, setup)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 907, in generative_auccess
    a  = sample_action_vector(ball, self.cache, uniform=self.uniform, radmode=self.radmode, mask=mask, check_border=True)
  File "/media/compute/homes/aharter/isy2020/phyre/phyre_utils.py", line 581, in sample_action_vector
    value = pic[ball].median()
RuntimeError: median cannot be called with empty tensor
2020-11-05 11:27:39,854 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-05 11:27:44,154 INFO __main__ VS-rad-median ball_within_template 0 || loading data for generative training...
2020-11-05 11:29:00,014 INFO __main__ VS-rad-median ball_within_template 0 || training 'generative' models...
2020-11-05 11:29:01,880 INFO root training epoch 1...
2020-11-05 11:29:23,374 INFO root collecting auccess after epoch 1...
2020-11-05 11:29:28,707 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-05 11:40:34,151 INFO root finish collecting auccess after epoch 1: 0.06924054109225995
2020-11-05 11:40:34,152 INFO root training epoch 2...
2020-11-05 11:40:55,162 INFO root collecting auccess after epoch 2...
2020-11-05 11:41:00,385 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-05 11:51:17,837 INFO root finish collecting auccess after epoch 2: 0.09704602804807237
2020-11-05 11:51:17,838 INFO root training epoch 3...
2020-11-05 11:51:38,630 INFO root training epoch 4...
2020-11-05 11:51:59,849 INFO root training epoch 5...
2020-11-05 11:52:21,046 INFO root collecting auccess after epoch 5...
2020-11-05 11:52:26,344 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
